wiki/versions/1.19/getting_started/setup/markdown/setup.md

```diff
@@ -0,0 +1,101 @@
+# Setup
+
+Quilt is a mod-loader toolchain project for Minecraft; it allows you to modify and expand upon the game.
+
+It is recommended that you have knowledge of Java, because it is the language which Minecraft is written in. <!--- TODO: Perhaps link to some Java resources and courses. -->
+
+## Setting Up your Workspace
+
+While you could create your own template, cloning the [official template mod](https://github.com/QuiltMC/quilt-template-mod)
+might be a bit simpler.
```

Comment:
The QTM also has a README that is pretty much a step-by-step on setting up a mod workspace, which might be good to point out here